packages/richtext-lexical/src/features/blocks/client/componentInline/index.tsx

```diff
@@ -116,6 +117,15 @@ export const InlineBlockComponent: React.FC<Props> = (props) => {
 
   const clientBlock = blocksField.blocks[0]
 
+  // Open drawer on mount
+  useEffect(() => {
+    // > 2 because they always have "id" and "blockName" fields
+    if (!hasMounted.current && clientBlock.fields.length > 2) {
```

Comment:
In another PR maybe we should also prevent `toggleDrawer` when clicking on the inline block if it has no fields